fastlane/lib/fastlane/actions/commit_version_bump.rb

```diff
@@ -58,8 +58,8 @@ def self.run(params)
         # Removes .plist files that matched the given expression in the 'ignore' parameter
         ignore_expression = params[:ignore]
         if ignore_expression
-          info_plist_files.select! do |info_plist_file|
-            !info_plist_file.match(ignore_expression)
+          info_plist_files.reject! do |info_plist_file|
```

Comment:
These changes are caused by `InverseMethod` cop.
https://github.com/bbatsov/rubocop/blob/c06b2e34d1a6108e03cd5c03761db28167005829/manual/cops_style.md#styleinversemethods

I think it should be enabled.